python/lsst/validate/drp/calcSrd.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+import numpy as np
+import scipy.stats
+
+# To validate these width estimates, we can plot a single random realization (re-evaluate the cell or call the function again to get a new one).
+def calcPA1(gv, magKey):
+    diffs = gv.aggregate(getRandomDiff, field=magKey)
+    means = gv.aggregate(np.mean, field=magKey)
+    rmsPA1, iqrPA1 = computeWidths(diffs)
+    return rmsPA1, iqrPA1, diffs, means
+
+
+def calcPA2(gv, magKey):
+    """Calculate PA2 values for min, design, stretch values of PF1.
```

Comment:
While there is a docstring here, I have similar comments as on `calcPA1`, above.